src/js/timeline.js

```diff
@@ -1,13 +1,56 @@
+function createTweetObj(obj){
+	var img = document.createElement('img');
+	img.setAttribute('src', obj.profileImageUrl);
+	var aTag = document.createElement('a');
+	aTag.setAttribute("target", "_blank");
+	aTag.setAttribute('href', "https://twitter.com/" + obj.twitterHandle + "/status/" + obj.statusId);
+	aTag.innerHTML = obj.message;
+	var span = document.createElement('span');
+	span.append(img);
+	span.append(aTag);
+	span.appendChild(document.createTextNode(new Date(obj.createdAt)));
+	return span;
+}
+
+
+
+
 function getTimeline(){
 	
 	var xhttp = new XMLHttpRequest();
 	var URL = "http://localhost:8080/api/1.0/twitter/timeline";
+	
+
     xhttp.onreadystatechange = function() {
          if (this.readyState == XMLHttpRequest.DONE && this.status == 200) {
-            document.getElementById('timelinePlaceholder').innerHTML = this.responseText; 
+	    
+			var div;
+         	var element = document.getElementById('timelinePlaceholder');
+         	element.innerHTML = "\n";
+          	var parsedJSON = JSON.parse(this.response);
+            for(var i = 0; i < parsedJSON.length; i++){
```

Comment:
separate out the ajax related code and the rendering related code. It should be easy to determine what code is rendering vs making the API call